pandas/tests/series/test_unary.py

```diff
@@ -55,3 +55,42 @@ def test_abs_nullable_int(self, any_signed_nullable_int_dtype, source, target):
         result = abs(ser)
         expected = Series(target, dtype=dtype)
         tm.assert_series_equal(result, expected)
+
```

Comment:
is it necessary to test again with series? since it delegates to the array

Reply:
I thought so too. When checking the implementation for integers, it tests in both array and series so I thought I was missing something and just followed suit.

Reply:
okay then let's follow the integer EA case

these tests could be consolidated too, i think (and possibly merged with the integer tests?)

Reply:
added another fixture and consolidated all the unary operator tests. I also consolidated the tests for integers